src/Tools/Extensions.ApiDescription.Server/src/build/Microsoft.Extensions.ApiDescription.Server.targets

```diff
@@ -61,6 +61,10 @@
       <_Command>$(_Command) $(OpenApiGenerateDocumentsOptions)</_Command>
     </PropertyGroup>
 
+    <PropertyGroup>
+      <OpenApiGenerateDocumentsOptions>$(OpenApiGenerateDocumentsOptions) --environment $(OpenApiGenerateEnvironment)</OpenApiGenerateDocumentsOptions>
```

Comment:
- If `$(OpenApiGenerateEnvironment)` has no value this will generate a switch with no value.
- If `$(OpenApiGenerateEnvironment)` contains spaces, the argument isn't correctly quoted.

Reply:
Thanks @martincostello, I have updated the `OpenApiGenerateDocumentsOptions` property to only include the environment parameter if `$(OpenApiGenerateEnvironment)` is not empty and quoted the `$(OpenApiGenerateEnvironment)` value to handle cases where it contains spaces. Please let me know if this is sufficient.